README.md

```diff
@@ -4,9 +4,12 @@ Rust implementation of JSON-RPC 2.0 Specification.
 Transport-agnostic `core` and transport servers for `http`, `ipc`, `websockets` and `tcp`.
 
 [![Build Status][travis-image]][travis-url]
+[![Build Status][appveyor-image]][appveyor-url]
 
 [travis-image]: https://travis-ci.org/paritytech/jsonrpc.svg?branch=master
 [travis-url]: https://travis-ci.org/paritytech/jsonrpc
+[appveyor-image]: https://ci.appveyor.com/api/projects/status/github/devops-parity/jsonrpc?svg=true
```

Comment:
@ordian this one actually should be `paritytech` since it's a github repo, not the account running the code (you can see that `paritytech` build is failing, while `devops-parity` is in `unknown` state.